### DISCUSSION TITLE
GitHub education pack verification

### DISCUSSION THREAD

**Discussion by dhwon0916** [community/community#52468] [GitHub Education]
I tried to reverify the academic status for GitHub educational pack. I tried uploading my transcript, dated schedule, etc., but it keeps returning the message:

Your application cannot be reviewed until you fix the following:
   - Your image should be one of the following: 
     Dated school ID
     
My school didn't provide us with a dated school ID, so I can't choose that option. My school also didn't give an email address that I can verify, so I can't use that. I tried all the suggestions on the document website, but it still doesn't work. Please help

**Comment by samus-aran** [comment]
Hi @dhwon0916 

You can submit multiple pieces of evidence in a single application. If you don't have a dated student ID take a photo of that on top of a transcript of your course as an example. Ensure you select the most appropriate **evidence type** in the dropdown. The order of options indicates the preference of evidence with Academic Email being at the top. If you don't have an Academic email or Student ID, do try and acquire evidence that is positioned higher rather than lower in that option list.

Hope this helps! Let us know how you get along 👍

**Reply by dhwon0916** [reply]
I tried all other options. I tried requesting 18 times. It always returns with this message:
![image](https://user-images.githubusercontent.com/49349808/231259693-771f0c0e-8778-48b5-a5bb-3d5dea9f89d4.png)

My school didn't give me a dated school ID, nor did they give me an academic email that I could verify. I also tried reaching out to the GitHub support, but they didn't reply.
